packet/loadbalancers.go

```diff
@@ -274,19 +274,25 @@ func (l *loadBalancers) reconcileServices(svcs []*v1.Service, remove bool) error
 }
 
 // addNodePeer update the configmap to ensure that the given node has the given peer
-func addNodePeer(config *metallb.ConfigFile, nodeName string, peer string, localASN, peerASN int) *metallb.ConfigFile {
+func addNodePeer(config *metallb.ConfigFile, nodeName string, peers []string, localASN, peerASN int) *metallb.ConfigFile {
```

Comment:
minor nit, but if changing the signature already, might make sense to make peers a variadic argument instead:
```suggestion
func addNodePeer(config *metallb.ConfigFile, nodeName string, localASN, peerASN int, peers ...string) *metallb.ConfigFile {
```

Reply:
Yeah, I can do that. I thought about it and was lazy. I will do it now.

Reply:
Updated.